exercises/sublist/sublist.py

```diff
@@ -1,2 +1,10 @@
-def check_lists(first_list, second_list):
+# Possible sublist categories.
+# Change the values as you see fit.
```

Comment:
These mirror the stub in the [yacht](https://github.com/exercism/python/blob/master/exercises/yacht/yacht.py) exercise.